templates/login.hbs

```diff
@@ -19,6 +19,10 @@
     <input type="hidden" name="{{@key}}" value="{{this}}" />
   {{/each}}
 
+  <p class="reset-password">
+    <a href="/reset-password">Reset Password</a>
```

Comment:
Lets make the label 'Forgot password'.

Also, this setting should be inside of the `login-options` `<p>` tag, so they are on the same line with the same style.